truestbyheart/politico
# Signin

#### What does this PR do?

 Centers the sign in page and adds label the input fields.

#### Description of Task to be completed?
All the misaligned  elements on the page where translated using relative unities so as to reduce the load when implementing  RWD
#### How should this be manually tested?
 - clone the repo to your local machine
 - once it cloned `cd` into UI.
 - double click on the `index.html` file to see the changes 
#### Any background context you want to provide?
 The element where misaligned and the inputs didn't have any labeling.
#### What are the relevant pivotal tracker stories?
#163865474
